ui/decredmaterial/icon.go

```diff
@@ -7,7 +7,7 @@ import (
 	"image"
 	"image/color"
 	"image/draw"
-	_ "image/png"
+	_ "image/png" // fixes unknown format error when trying to decode png images
```

Comment:
this comment should be rephrased. It makes png a decodable format, it doesn't fix an error.